### DISCUSSION TITLE
SVG file won´t open in inkscape or Adobe illustrater

### DISCUSSION THREAD

**Discussion by coz29** [motoz/PellMon#113] [General]
Hey Pellmon. 

I was looking at the custom image for pellmon and followed your guide to use inkscape but it doesn't open the file in inkscape or adobe illustrator. What to do about it?

**Comment by motoz** [comment]
Please explain step by step how you are trying to open the file because I can open them just fine in inkscape (0.92) and I can't figure out what might be the problem.

**Reply by coz29** [reply]
I start Inkscape and choose File in the top and open file and choose the SVG file and I get the error that the program not can read the file, same problem ind Adobe illustrator. I have also tried to import the file to the program same error.

**Reply by motoz** [reply]
How did you get the svg file, git clone or download as zip or something else? It's really strange because File->Open just works for me as it always has.

**Reply by coz29** [reply]
I have downloaded from pellmon image site https://github.com/motoz/PellMon/tree/master/src/Pellmonweb/media/img

**Reply by coz29** [reply]
I found a solution. I downloaded the github pellmon master file, now all works. 

Is it possible to add water external return temperature sensor to pellmon via the raspberry pi gpio inputs or mqtt input ?

**Reply by motoz** [reply]
Great! It still works for me if I click on an svg file from that link and then right click on the image and choose 'save as', but you can also just download the whole project as you did or clone it.
